### PR TITLE
Use 11ty to deploy images/resources

### PR DESCRIPTION
**Pull request by agners**
This allows to add images in the source directory 11ty will deploy them without restarting script/develop.

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -2,6 +2,12 @@ const yaml = require("js-yaml");
 
 module.exports = function (eleventyConfig) {
   eleventyConfig.addDataExtension("yaml", (contents) => yaml.load(contents));
+  eleventyConfig.addPassthroughCopy(
```
Comment (ludeeus) [inline .eleventy.js:5]: Cant, we remove `serve` with this change now?
Everything should not work with `eleventy --serve --watch` directly right?

Comment (agners) [reply .eleventy.js:5]: Oh, wasn't aware of the `--serve` option. Yeah I think that should work, let me give this a try.

Comment (agners) [reply .eleventy.js:5]: Works :+1:

**Review by ludeeus — APPROVED** [review]
Nice 👍

**Pull request merged**